# translation of uk(5).po to Ukrainian
# Maxim Dziumanenko <mvd@mylinux.ua>, 2005.
# Vadim Abramchuck <Abram@email.ua>, 2006.
# Ukrainian translation of update-manager.
# Copyright (C) 2005, 2006 Free Software Foundation, Inc.
#: ../DistUpgrade/DistUpgradeController.py:1075
msgid ""
msgstr ""
"Project-Id-Version: uk(5)\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-08-26 08:38-0700\n"
"PO-Revision-Date: 2019-07-12 15:58+0000\n"
"Last-Translator: Юрій Олексійчук <Unknown>\n"
"Language-Team: Ukrainian <uk@li.org>\n"
"Language: uk\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:48+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Сервер для %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Основний сервер"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Сервери користувача"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "Неможливо підрахувати запис sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Не вдалося знайти жодного файлу пакунків; можливо, це не диск Ubuntu або "
"диск для іншої архітектури?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "Не вдалося додати компакт-диск"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Виникла помилка при додаванні компакт-диску, оновлення буде припинено. Будь-"
"ласка, повідомте про цю помилку, якщо це правильний компакт-диск Ubuntu.\n"
"Повідомлення про помилку:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Вилучити пакунок, що знаходиться в поганому стані"
msgstr[1] "Вилучити пакунки, що знаходяться в поганому стані"
msgstr[2] "Вилучити пакунки, що знаходяться в поганому стані"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Пакунок '%s' знаходиться у несумісному стані і повинен бути "
"перевстановлений, але не знайдено жодного архіву для нього. Чи ви бажаєте "
"видалити цей пакунок зараз, щоб продовжити?"
msgstr[1] ""
"Пакунки'%s' знаходяться у несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Чи ви бажаєте видалити ці пакунки "
"зараз, щоб продовжити?"
msgstr[2] ""
"Пакунки '%s' знаходяться у несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Чи ви бажаєте видалити ці пакунки "
"зараз, щоб продовжити?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "Схоже, що сервер перенавантажений"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Пошкоджені пакунки"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Ваша система містить пошкоджені пакунки, котрі не можуть бути виправлені "
"цією програмою. Будь-ласка, виправіть їх програмами synaptic або apt-get."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Пошкоджені пакунки"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""
"Під час розрахунку оновлення виникла нерозв'язувана проблема.\n"
"\n"
" "

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""
"Ймовірно, причиною є:\n"
" * Неофіційні пакунки із програмним забезпеченням, які не є частиною Ubuntu\n"
"Будь ласка, скористайтеся програмою ppa-purge з пакунка ppa-purge \n"
"для вилучення програмного забезпечення з PPA Launchpad, а потім \n"
"повторіть спробу оновлення.\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""
"Причина:\n"
" * Оновлення до тестової версії Ubuntu\n"
"Ймовірно, проблема є тимчасовою. \n"
"Будь ласка, повторіть спробу пізніше.\n"

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr "Перелік змін ще не готовий, спробуйте пізніше."

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""
"Якщо жодне із цих припущень є не правильним, будь ласка, повідомте про цю "
"ваду за допомогою команди «ubuntu-bug ubuntu-release-upgrader-core», "
"віддавши її у терміналі. "

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""
"Якщо захочете розібратися із причинами власноруч, у файлах журналу у /var/"
"log/dist-upgrade міститимуться подробиці щодо оновлення. Особливу увагу "
"зверніть на main.log та apt.log."

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "Не вдалося розрахувати оновлення"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "Помилка аутенфікації деяких пакетів"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Не вдалося перевірити деякі пакети. Це може бути викликано проблемами в "
"мережі. Можливо, Ви захочете спробувати пізніше. Список не перевірених "
"пакетів надано нижче."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Пакунок '%s' помічено для видалення, але він в чорному списку видалення."

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Важливий пакунок '%s' помічено для видалення."

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Спроба встановити версію '%s' чорного списку"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "Не можливо встановити '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Не вдалося встановити потрібний пакунок. Повідомте про цю помилку, "
"використавши команду 'ubuntu-bug ubuntu-release-upgrader-core' в терміналі."

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "Не можливо підібрати meta-пакунок"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Ваша система не містить такого %s чи такого %s пакунку, було неможливо "
"визначити, якою версією Убунту ви користуєтеся.\n"
" Будь ласка, встановіть один з пакунків вище за допомогою synaptic або apt-"
"get, перед тим як продовжити."

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "Зчитування кешу"

#: ../DistUpgrade/DistUpgradeController.py:225
msgid "Unable to get exclusive lock"
msgstr "Не вдалося отримати ексклюзивне блокування"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Це зазвичай значить, що запущено інший менеджер пакунків (наприклад, apt-get "
"або aptitude). Будь ласка, спочатку закрийте цю програму."

#: ../DistUpgrade/DistUpgradeController.py:284
msgid "Upgrading over remote connection not supported"
msgstr "Віддалене оновлення системи більше не підтримується"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Ви намагаєтесь виконати оновлення через ssh-з'єднання з непідтримуваним "
"клієнтом. Оновіться у текстовому режимі через \"do-release-upgrade\".\n"
"\n"
"Оновлення зупинене. Спробуйте без ssh."

#: ../DistUpgrade/DistUpgradeController.py:299
msgid "Continue running under SSH?"
msgstr "Продовжити роботу через SSH?"

#: ../DistUpgrade/DistUpgradeController.py:300
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Ця сесія, скоріш за все, працює через ssh. Наполегливо рекомендуємо не "
"виконувати оновлення через ssh через те, що в разі невдачі буде важче "
"відновитись.\n"
"\n"
"Якщо Ви продовжите, буде запущено додаткову службу ssh на порті '%s'.\n"
"Чи бажаєте продовжити?"

#: ../DistUpgrade/DistUpgradeController.py:314
msgid "Starting additional sshd"
msgstr "Запуск додаткового sshd"

#: ../DistUpgrade/DistUpgradeController.py:315
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Щоб зробити відновлення легшим у разі поломки, на порті '%s' буде запущений "
"додатковий sshd. Якщо щось піде не так із запуском ssh, Ви все ще зможете "
"під'єднатись до другого.\n"

#: ../DistUpgrade/DistUpgradeController.py:323
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Якщо брандмауер увімкнений, треба тимчасово відкрити цей порт. Це потенційно "
"небезпечно, тому потребує втручання користувача. Ви можете відкрити порт "
"наступним чином:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:401
#: ../DistUpgrade/DistUpgradeController.py:425
msgid "Can not upgrade"
msgstr "Оновлення неможливе"

#: ../DistUpgrade/DistUpgradeController.py:402
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Оновлення з '%s' до '%s' цією програмою не підтримується."

#: ../DistUpgrade/DistUpgradeController.py:426
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""
"Інсталяція python3 install пошкоджена. Виправте символьне посилання '/usr/"
"bin/python3'."

#: ../DistUpgrade/DistUpgradeController.py:455
#, python-format
msgid "Can not write to '%s'"
msgstr "Неможливо виконати запис в '%s'"

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Неможливо виконати запис в системний каталог '%s' у Вашій системі. Оновлення "
"не може бути продовженим.\n"
"Переконайтеся, що системний каталог доступний для запису."

#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Include latest updates from the Internet?"
msgstr "Додати останні оновлення з Інтернету?"

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Система оновлення може використовувати Інтернет для автоматичного "
"завантаження останніх оновлень та встановлювати їх під час оновлення "
"системи. Якщо у вас є мережеве підключення — це наполегливо рекомендується.\n"
"\n"
"Оновлення буде більш тривалим, проте коли завершиться, Ваша система буде "
"повністю оновлена. Ви можете відмовитись від цього, але ви маєте встановити "
"останні оновлення одразу після завершення процесу оновлення.\n"
"Якщо Ви тут відповісте 'ні', мережа не буде використовуватись взагалі."

#: ../DistUpgrade/DistUpgradeController.py:646
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr "Не для людей, поки знаходиться на стадії розробки релізу %s"

#: ../DistUpgrade/DistUpgradeController.py:747
#, python-format
msgid "disabled on upgrade to %s"
msgstr "відключено під час оновлення до %s"

#: ../DistUpgrade/DistUpgradeController.py:806
msgid "No valid sources.list entry found"
msgstr "Не знайдено жодних дійсних записів source.list"

#: ../DistUpgrade/DistUpgradeController.py:807
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"Під час сканування даних вашого сховища неможливо знайти запис про %s.\n"
"\n"
"Оновлення може не відбутися.\n"
"\n"
"Продовжувати в будь-якому випадку?"

#: ../DistUpgrade/DistUpgradeController.py:817
msgid "No valid mirror found"
msgstr "Не знайдено правильного дзеркала"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"При скануванні сховища не було знайдено дзеркало для оновлення. Таке могло "
"статися у випадку використання внутрішнього дзеркала або інформація про "
"дзеркала застаріла.\n"
"\n"
"Все одно бажаєте переписати файл джерел 'sources.list'? Якщо ви вкажете "
"\"Так, тоді оновляться усі записи від '%s' до '%s'.\n"
"Якщо вкажете \"Ні\" оновлення буде припинено."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Generate default sources?"
msgstr "Створити типові джерела?"

#: ../DistUpgrade/DistUpgradeController.py:839
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Після сканування вашого 'sources.list' не було знайдено дійсного запису для "
"'%s'.\n"
"\n"
"Додати типовий запис для '%s'? Якщо ви оберете 'Ні', оновлення буде "
"скасовано."

#: ../DistUpgrade/DistUpgradeController.py:874
#: ../DistUpgrade/DistUpgradeController.py:880
msgid "Repository information invalid"
msgstr "Помилка в даних про сховище"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Оновлення інформації про репозиторій закінчилося невдачею (недійсний файл), "
"тому запустився процес звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:881
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Спроба оновлення даних щодо сховища призвела до спотворення даних у файлі. "
"Щоб повідомити про ваду, встановіть apport, а потім віддайте команду «apport-"
"bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:890
msgid "Third party sources disabled"
msgstr "Сторонні джерела відключені"

#: ../DistUpgrade/DistUpgradeController.py:891
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Деякі сторонні джерела у Вашому sources.list були відключені. Ви можете "
"знову включити їх після оновлення за допомогою програми 'software-"
"properties' чи менеджера пакунків."

#: ../DistUpgrade/DistUpgradeController.py:933
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Пакунок у незадовільному стані"
msgstr[1] "Пакунки у незадовільному стані"
msgstr[2] "Пакунків у незадовільному стані"

#: ../DistUpgrade/DistUpgradeController.py:936
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Пакунок '%s' знаходиться в несумісному стані і повинен бути "
"перевстановлений, але не знайдено жодного архіву для нього. Будь ласка, "
"перевстановіть пакунок вручну чи вилучіть його з системи."
msgstr[1] ""
"Пакунки '%s' знаходяться в несумісному стані, було б добре їх "
"перевстановити, та не вдається знайти для них ніяких архівів. Будь ласка, "
"перевстановіть пакунки вручну чи вилучіть їх із системи."
msgstr[2] ""
"Пакунки '%s' знаходяться в несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Будь ласка, перевстановіть пакунки "
"вручну чи вилучіть їх з системи."

#: ../DistUpgrade/DistUpgradeController.py:999
msgid "Error during update"
msgstr "Помилка під час оновлення"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Під час оновлення сталася помилка. Це зазвичай викликано проблемами у "
"мережі, будь ласка, перевірте Ваше мережеве з’єднання та спробуйте знову."

#: ../DistUpgrade/DistUpgradeController.py:1043
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "EFI System Partition (ESP) not usable"
msgstr "Системний розділ EFI (ESP) не підходить для використання"

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"Системний розділ EFI (ESP) не змонтований на /boot/efi. Переконайтеся, що "
"він належним чином налаштований і спробуйте ще раз."

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"Системний розділ EFI (ESP), що змонтований на /boot/efi недоступний для "
"запису. Змонтуйте його у режим читання-запису і спробуйте ще раз."

#: ../DistUpgrade/DistUpgradeController.py:1056
msgid "Not enough free disk space"
msgstr "Недостатньо місця на диску"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1059
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"Оновлення перервано. Для здійснення оновлення потрібно {str_total} вільного "
"місця на диску \"{str_dir}\". Будь ласка звільніть принаймні ще {str_needed} "
"дискового простору на \"{str_dir}\". {str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""
"Видаліть тимчасові пакунки попередніх інсталяцій,  виконавши команду 'sudo "
"apt clean'."

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"Ви можете видалити старі ядра за допомогою команди «sudo apt autoremove» а "
"також додати COMPRESS=xz до файла /etc/initramfs-tools/initramfs.conf для "
"зменшення розмірів initramfs."

#: ../DistUpgrade/DistUpgradeController.py:1071
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Очистіть кошик та видаліть тимчасові пакунки попередніх інсталяцій за "
"допомогою команди «sudo apt-get clean»."

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid "Reboot to clean up files in /tmp."
msgstr "Для очищення файлів у  /tmp , необхідне перезавантаження."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1113
#: ../DistUpgrade/DistUpgradeController.py:1989
#: ../DistUpgrade/DistUpgradeController.py:2039
msgid "Calculating the changes"
msgstr "Проводиться аналіз змін"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Do you want to start the upgrade?"
msgstr "Бажаєте почати оновлення системи?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""
"Оновлення захисту Livepatch для Ubuntu %s є недоступними. Якщо ви оновите "
"систему, Livepatch буде вимкнено."

#: ../DistUpgrade/DistUpgradeController.py:1248
msgid "Upgrade canceled"
msgstr "Оновлення скасовано"

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Оновлення буде скасовано та систему буде відновлено до попереднього стану. "
"Ви зможете продовжити оновлення пізніше."

#: ../DistUpgrade/DistUpgradeController.py:1255
#: ../DistUpgrade/DistUpgradeController.py:1400
msgid "Could not download the upgrades"
msgstr "Неможливо завантажити пакунки для оновлення системи"

#: ../DistUpgrade/DistUpgradeController.py:1256
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Оновлення було перервано. Будь ласка, перевірте ваше інтернет-з'єднання чи "
"носій, з якого здійснюється встановлення, та спробуйте знову. Всі "
"завантажені файли будуть збережені."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1349
#: ../DistUpgrade/DistUpgradeController.py:1388
#: ../DistUpgrade/DistUpgradeController.py:1504
msgid "Error during commit"
msgstr "Помилка при фіксації"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1351
#: ../DistUpgrade/DistUpgradeController.py:1390
#: ../DistUpgrade/DistUpgradeController.py:1543
msgid "Restoring original system state"
msgstr "Відновлення первісного стану системи"

#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1367
#: ../DistUpgrade/DistUpgradeController.py:1391
msgid "Could not install the upgrades"
msgstr "Неможливо провести оновлення системи"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1357
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Оновлення було перервано. Ваша система може бути в нестабільному стані. "
"Зараз буде запущено процес відновлення (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1362
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Будь ласка, повідомте про цю помилку на сторінці за адресою http://bugs."
"launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug, вклавши файли "
"з /var/log/dist-upgrade/ до звіту про помилку.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1401
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Оновлення було перервано. Будь ласка, перевірте Ваше з'єднання з Інтернетом "
"чи носій для встановлення та спробуйте знову. "

#: ../DistUpgrade/DistUpgradeController.py:1429
msgid "Searching for obsolete software"
msgstr "Пошук застарілих програм"

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Remove obsolete packages?"
msgstr "Видалити застарілі пакунки?"

#: ../DistUpgrade/DistUpgradeController.py:1493
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Залишити"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "_Remove"
msgstr "Видалити"

#: ../DistUpgrade/DistUpgradeController.py:1505
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"При очищенні системи виникли проблеми. Прочитайте детальнішу інформацію "
"нижче. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Required depends is not installed"
msgstr "Необхідний пакет (залежність) не встановлений"

#: ../DistUpgrade/DistUpgradeController.py:1582
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Необхідний пакет (залежність) '%s' не встановлений. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1849
#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "Checking package manager"
msgstr "Перевірка програми управління пакунками"

#: ../DistUpgrade/DistUpgradeController.py:1855
#: ../DistUpgrade/DistUpgradeController.py:1861
msgid "Preparing the upgrade failed"
msgstr "Підготовка до оновлення не вдалася"

#: ../DistUpgrade/DistUpgradeController.py:1856
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Підготовка системи до оновлення закінчилася невдало. Почато процес "
"звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:1862
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Підготування системи до оновлення закінчилося невдачею. Щоб повідомити про "
"помилку, встановіть apport та виконайте команду 'apport-bug ubuntu-release-"
"upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:1879
#: ../DistUpgrade/DistUpgradeController.py:1889
msgid "Getting upgrade prerequisites failed"
msgstr "Помилка отримання необхідних файлів для оновлення"

#: ../DistUpgrade/DistUpgradeController.py:1880
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Системі не вдалося отримати попередні конфігурації для повного оновлення. "
"Зараз процес оновлення буде перервано і буде відновлено початковий стан "
"системи.\n"
"\n"
"Також почато процес звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:1890
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"Система не змогла отримати потрібні умови для оновлення. Оновлення буде "
"зараз перевано й відновиться початковий стан системи.\n"
"\n"
"Повідомити про помилку встановлення apport й потім виконати 'apport-bug "
"ubuntu-release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:1923
msgid "Updating repository information"
msgstr "Оновлення інформації про сховище"

#: ../DistUpgrade/DistUpgradeController.py:1930
msgid "Failed to add the cdrom"
msgstr "Помилка при підключенні CD-ROM"

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Sorry, adding the cdrom was not successful."
msgstr "На жаль, не вдалося підключити CD-ROM"

#: ../DistUpgrade/DistUpgradeController.py:1966
msgid "Invalid package information"
msgstr "Невірна інформація про пакунок"

#: ../DistUpgrade/DistUpgradeController.py:1967
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Після оновлення інформації про пакунок не вдалося знайти важливий пакунок "
"'%s'. Причиною цього може бути відсутність офіційних дзеркал у вашому списку "
"джерел програмного забезпечення або тому, що дзеркало, яке ви "
"використовуєте, перевантажено. Див. поточний список налаштованих джерел "
"програмного забезпечення в /etc/apt/sources.list.\n"
"Якщо дзеркало перевантажено, можна знову спробувати оновитися згодом."

#: ../DistUpgrade/DistUpgradeController.py:1996
#: ../DistUpgrade/DistUpgradeController.py:2095
msgid "Fetching"
msgstr "Отримання"

#: ../DistUpgrade/DistUpgradeController.py:2003
#: ../DistUpgrade/DistUpgradeController.py:2024
#: ../DistUpgrade/DistUpgradeController.py:2045
#: ../DistUpgrade/DistUpgradeController.py:2099
msgid "Upgrading"
msgstr "Процес оновлення"

#: ../DistUpgrade/DistUpgradeController.py:2005
msgid "Upgrade infeasible"
msgstr "Оновлення неможливе"

#: ../DistUpgrade/DistUpgradeController.py:2006
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr "Неможливо виконати оновлення, під час оновлення були помилки."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2027
msgid "Upgrade incomplete"
msgstr "Оновлення неповне"

#: ../DistUpgrade/DistUpgradeController.py:2028
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""
"Оновлення частково завершено, але під час процесу оновлення були помилки."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2050
#: ../DistUpgrade/DistUpgradeController.py:2101
#: ../DistUpgrade/DistUpgradeController.py:2107
#: ../DistUpgrade/DistUpgradeController.py:2118
msgid "Upgrade complete"
msgstr "Оновлення завершено"

#: ../DistUpgrade/DistUpgradeController.py:2051
#: ../DistUpgrade/DistUpgradeController.py:2102
#: ../DistUpgrade/DistUpgradeController.py:2108
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Оновлення було завершено, але в його процесі виникли деякі помилки."

#: ../DistUpgrade/DistUpgradeController.py:2070
msgid "System upgrade is complete."
msgstr "Оновлення системи завершено."

#: ../DistUpgrade/DistUpgradeController.py:2119
msgid "The partial upgrade was completed."
msgstr "Часткове оновлення завершене."

#: ../DistUpgrade/DistUpgradeController.py:2147
msgid "Lock screen disabled"
msgstr "Екран блокування вимкнено"

#: ../DistUpgrade/DistUpgradeController.py:2148
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr "Екран блокування вимкнено до наступного перевантаження системи."

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Не вдалося знайти примітки випуску."

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Сервер може бути перенавантажений. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Не вдалося завантажити примітки випуску."

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Будь ласка, перевірте ваше з'єднання з Інтернетом."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "автентифікувати '%(file)s' замість '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "видобування '%s'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "Неможливо провести апргрейд системи"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Найімовірнішою причиною може бути помилка в інструменті оновлення. Повідомте "
"про цю помилку за допомогою команди 'ubuntu-bug ubuntu-release-upgrader-"
"core'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "Підпис інструменту оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "Інструмент оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "Не вдалося отримати"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Не вдалося отримати оновлення. Це може бути викликано проблемою у мережі. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "Помилка аутентифікації"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Автентифікація оновлення зазнала краху. Це може бути викликано проблемою з "
"мережею або сервером. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "Не вдалося розпакувати"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Не вдалося розпакувати оновлення. Можливо, виникла проблема в мережі або на "
"сервері. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "Перевірка не вдала"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Не вдалося перевірити наявність оновлень. Можливо, виникла проблема в мережі "
"або на сервері. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "Не вдається почати оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Це, зазвичай, виникає в системах з /tmp змонтованому з noexec. Перемонтуйте, "
"будь-ласка, без noexec та виконайте оновлення знов."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "Повідомлення про помилку: '%s'."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Оновити"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&Скасувати"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Відомості про релізе"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Оновлення"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Завантаження додаткових файлів пакунків..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Файл %s з %s при %sБ/с"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Файл %s з %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Вставте '%s' в привід '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Зміна носія"

#: ../DistUpgrade/DistUpgradeQuirks.py:230
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Ваше графічне обладнання не може бути повністю підтримуване в Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:232
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"Середовище робочого столу 'unity' підтримується вашим графічним апаратним "
"забезпеченням не повністю. Можливо, після оновлення швидкодія середовища "
"буде сповільнена. Радимо на разі залишити версію LTS. Щоб отримати докладні "
"відомості, див. https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Ви все ще хочете продовжити оновлення?"

#: ../DistUpgrade/DistUpgradeQuirks.py:256
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "Ваш графічний пристрій підтримується Ubuntu 12.04 LTS не повністю."

#: ../DistUpgrade/DistUpgradeQuirks.py:258
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Підтримку Вашого графічного адаптера Intel в Ubuntu 12.04 LTS обмежено, і Ви "
"можете стикнутися з проблемами після оновлення. Для отримання детальної "
"інформації відвідайте https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForI8xx Ви дійсно бажаєте продовжити оновлення?"

#: ../DistUpgrade/DistUpgradeQuirks.py:278
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Оновлення може уповільнити ефекти робочого столу, продуктивність роботи в "
"іграх та інших графічно складних програмах."

#: ../DistUpgrade/DistUpgradeQuirks.py:282
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Цей комп'ютер зараз використовує графічний драйвер AMD 'fglrx'. Нема "
"доступних версій драйверів, які б працювали з вашим обладнанням на Ubuntu "
"10.04 LTS.\n"
"\n"
"Чи хочете ви продовжувати?"

#: ../DistUpgrade/DistUpgradeQuirks.py:321
msgid "Sorry, no more upgrades for this system"
msgstr "Вибачте, цю систему вже не можна оновити"

#: ../DistUpgrade/DistUpgradeQuirks.py:322
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""
"Подальших випусків Ubuntu для архітектури «i386» цієї системи не буде.\n"
"\n"
"Пакунки оновлень Ubuntu %s випускатимуть до %s."

#: ../DistUpgrade/DistUpgradeQuirks.py:337
msgid "No ARMv6 CPU"
msgstr "Немає ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:338
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ваша система використовує процесор ARM з архітектурою, старішою за ARMv6. "
"Всі пакети у karmic були скомпільовані з оптимізаціями, які вимагають ARMv6 "
"як мінімум. Оновлення вашої системи до нового випуску Ubuntu неможливе на "
"даному обладнанні."

#: ../DistUpgrade/DistUpgradeQuirks.py:358
msgid "No init available"
msgstr "Служба init недоступна"

#: ../DistUpgrade/DistUpgradeQuirks.py:359
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Схоже, що ваша система є віртуалізованим оточенням без служби init, "
"наприклад Linux-VServer. Ubuntu 10.04 LTS не може працювати в такому типі "
"оточення, ви повинні спочатку виправити конфігурацію своєї віртуальної "
"машини.\n"
"\n"
"Ви впевнені, що хочете продовжити?"

#: ../DistUpgrade/DistUpgradeQuirks.py:400
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:401
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:519
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "Помилка з'єднання з магазином Snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:520
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:535
msgid "Connection to Snap Store failed"
msgstr "Помилка з'єднання з магазином Snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:536
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Ваша система не має підключення до Snap Store. Щоб отримати найкращу "
"процедуру оновлення, переконайтеся, що система може підключитися до api."
"snapcraft.io. Продовжити оновлення?"

#: ../DistUpgrade/DistUpgradeQuirks.py:545
msgid "Outdated snapd package"
msgstr "Застарілий пакунок snapd"

#: ../DistUpgrade/DistUpgradeQuirks.py:546
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"У системі відсутній snapd останньої версії. Будь ласка, оновіть версію snapd "
"у системі, щоб мати покращену процедуру оновлення. Продовжити оновлення?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:569
msgid "Calculating snap size requirements"
msgstr "Обчислюємо вимоги щодо розміру для snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:600
msgid "Processing snap replacements"
msgstr "Обробляємо замінники snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:606
#, python-format
msgid "refreshing snap %s"
msgstr "оновлюємо snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:610
#, fuzzy, python-format
msgid "removing snap %s"
msgstr "оновлюємо snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:613
#, python-format
msgid "installing snap %s"
msgstr "встановлюємо snap %s"

#: ../DistUpgrade/DistUpgradeQuirks.py:642
msgid "PAE not enabled"
msgstr "PAE не увімкнуто"

#: ../DistUpgrade/DistUpgradeQuirks.py:643
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"У вашій системі використовується процесор, у якому не увімкнено інструкцій "
"PAE. Користуватися Ubuntu у системах без PAE можна лише до версії Ubuntu "
"12.10. Оновлення до новішої версії Ubuntu вимагатиме вмикання PAE (якщо це "
"можливо). Див.: http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:966
msgid "Checking for installed snaps"
msgstr "Перевірка на встановлені snap-пакунки"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Вживати наступний шлях для пошуку компакт-диска з пакунками оновлення"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Використовувати інтерфейс. В наявності: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*ЗАСТАРІЛЕ* Цю опцію буде проігноровано"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Виконати лише часткове оновлення (без перезапису sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "Вимкнути підтримку GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Вказати каталог з даними"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "Оновитися до випуску що розробляється"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "Отримання завершено"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Отримання файлу %li з %li на швидкості %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Приблизно '%s' залишилось"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Отримання файлу %li з %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Застосування змін"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "проблеми з залежностями - залишається неналаштованим"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "Не вдалося встановити '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Обновлення продовжиться, але пакет '%s' може бути в неробочому стані. Будь "
"ласка, надішліть звіт про помилку."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Замінити змінений користувачем конфігураційний файл\n"
"'%s'?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Ви втратите всі зроблені Вами у цьому конфігураційному файлі зміни якщо "
"заміните його новою версією."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "Команда 'diff' не знайдена"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Виникла невиправна помилка"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Будь ласка, повідомте про цю помилку (якщо ви ще це не зробили) та додайте "
"фали /var/log/dist-upgrade/main.log та /var/log/dist-upgrade/apt.log до "
"вашого звіту. Оновлення було перервано.\n"
"Ваш оригінальний sources.list було збережено в /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Натиснено Ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Операція буде скасована, що може залишити систему в несправному стані. Ви "
"все ще хочете це зробити?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr "Для запобігання втрати інформації закрийте усі програми та документи."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Більше не підтримується Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Встановлення старої версії (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Видалити (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Більш не потрібен (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Встановлення (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Оновлення (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr "Скасувати"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr "Далі"

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Показати відмінності >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Сховати відмінності"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Помилка"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Закрити"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Показати Термінал >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Сховати Термінал"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Інформація"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Почати оновлення"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Деталі"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Більш не підтримується %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "Видалити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Видалити (був автоматично інстальований) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "Встановити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "Оновити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Необхідне перезавантаження"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Перезавантажте систему для завершення оновлення</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "Перезапустити зараз"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Скасувати запущене оновлення?</big></b>\n"
"\n"
"Система може потрапити у непридатний стан, якщо скасувати оновлення. Вам "
"наполегливо рекомендується продовжити оновлення."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "Скасувати оновлення?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li день"
msgstr[1] "%li дні"
msgstr[2] "%li днів"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li година"
msgstr[1] "%li години"
msgstr[2] "%li годин"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li хвилина"
msgstr[1] "%li хвилини"
msgstr[2] "%li хвилин"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li секунда"
msgstr[1] "%li секунди"
msgstr[2] "%li секунд"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Це завантаження триватиме близько %s з 1Mбіт DSL-з'єднанням та біля %s з 56k "
"модемом."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Це завантаження триватиме приблизно %s. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Підготовка до оновлення"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Отримання нових каналів програм"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Отримання нових пакунків"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Встановлення оновлень"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Очищення"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d установлений пакет більш не підтримується Canonical. Ви можете "
"отримати підтримку у спільноти."
msgstr[1] ""
"%(amount)d установлених пакетів більш не підтримуються Canonical. Ви можете "
"отримати підтримку у спільноти."
msgstr[2] ""
"%(amount)d установлених пакетів більш не підтримуються Canonical. Ви можете "
"отримати підтримку у спільноти."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] ""
"Copy text   \t\r\n"
"%d пакет буде видалено."
msgstr[1] "%d пакети буде видалено."
msgstr[2] "%d пакетів буде видалено."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Буде встановлено %d новий пакет."
msgstr[1] "Буде встановлено %d нових пекети."
msgstr[2] "Буде встановлено %d нових пекетів."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "Буде оновлено %d пакунок."
msgstr[1] "Буде оновлено %d пакунки."
msgstr[2] "Буде оновлено %d пакунків."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Вам треба завантажити всього %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Встановлення оновлень може зайняти декілька годин. Після завершення "
"завантаження процес неможливо скасувати."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Отримання та встановлення оновлень може зайняти декілька годин. Після "
"завершення завантаження процес неможливо скасувати."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "Видалення пакунків може зайняти декілька годин. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "Програмне забезпечення на цьому комп’ютері не потребує оновлення."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "Немає оновлень для Вашої системи. Оновлення буде скасовано."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Необхідно перезавантажити систему"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Виконання оновлення завершено. Необхідно перезавантажити систему. "
"Перезавантажити зара?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Виконано: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Будь ласка, повідомте про цю помилку та додайте файли  /var/log/dist-upgrade/"
"main.log та /var/log/dist-upgrade/apt.log до вашого звіту. Оновлення було "
"перервано.\n"
"Ваш оригінальний sources.list було збережено в /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Перервано"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Понижено версію:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Щоб продовжити натисніть [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Продовжити [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Подробиці [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Більше не підтримується: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Видалити: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Встановити %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Оновити %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Продовжити [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Потрібне перезавантаження для завершення оновлення.\n"
"Оберіть 'y' для перезавантаження."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Завантаження файлу %(current)li з %(total)li із швидкістю %(speed)s/с"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Завантаження файлу %(current)li з %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Показувати прогрес для окремих файлів"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Скасувати оновлення"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "Продовжити оновлення"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Відмінити оновлення системи?</big></b>\n"
"\n"
"Зауважте, що якщо Ви скасуєте оновлення, це може призвести до нестабільного "
"стану системи. Дуже рекомендується продовжити оновлення."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Почати оновлення"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Замінити"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Різниця між файлами"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "Повідомити про помилку"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "Пр_одовжити"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Почати оновлення?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Перезавантажте Вашу систему для завершення оновлення</big></b>\n"
"\n"
"Будь ласка, збережіть Вашу дані перед продовженням."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Оновлення дистрибутивів"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>Оновлюємо Ubuntu до версії %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Встановлення нових каналів програм"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Перезавантаження системи"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Термінал"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Оновлення"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Доступна нова версія Ubuntu. Хочете оновити?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Не оновлювати"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Запитати пізніше"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Так, оновити зараз"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Ви відмовилось від оновлення Ubuntu до нової версії"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Можна виконати оновлення пізніше, відкривши Оновлювач програм і натиснувши "
"\"Оновити\"."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Виконати оновлення випуску"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Для оновлення Ubuntu, потрібна автентифікація."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Виконати часткове оновлення"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Для часткового оновлення, ви повинні пройти автентифікацію"

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Показати версію та вийти"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Каталог, який вміщує файли даних."

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Запустити вказаний інтерфейс оболонки"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Проводиться часткове оновлення"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Завантаження засобу оновлення випуску"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""
"Якщо використовується найсвіжіший підтримуваний випуск, оновитися до "
"випуску, що перебуває у розробці"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr "Спробуйте оновитись до останньої версії за допомогою $distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Запустити в особливому режимі оновлення.\n"
"Наразі підтримуються режим \"стільниця\" для регулярного оновлення "
"персональних робочих станцій та \"сервер\" для серверних систем."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Перевіряти доступність нової версії дистрибутива і повертати результат за "
"допомогою кода виходу."

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""
"Спробуйте оновлення із увімкненими сторонніми дзеркалами та сховищами, "
"замість спроби із закоментованими записами дзеркал і сховищ."

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "Параметри --devel-release і --proposed"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr ""
"не можна використовувати одночасно. Будь ласка, використовуйте лише один з "
"них."

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "Перевіряється наявність нового випуску Ubuntu"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ваша версія Ubuntu вже не підтримується."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Для отримання інформації щодо оновлення, будь ласка, відвідайте:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr "У /etc/update-manager/release-upgrades Prompt "

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr "встановлено на ніколи, тому оновлення неможливе."

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr "Версії, яка перебувала б у розробці, для LTS не існує."

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""
"Щоб оновити систему до найсвіжішого випуску, який не є випуском LTS і "
"перебуває у розробці, "

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""
"встановіть значення Prompt=normal у /etc/update-manager/release-upgrades."

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr "Оновлення до випуску, що перебуває у розробці, доступні "

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr "лише для найсвіжішого підтримуваного випуску."

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr "Нових випусків не знайдено."

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "Оновлення реліза зараз неможливо."

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Оновлення реліза не може бути виконано зараз, будь ласка, спробуйте пізніше. "
"Сервер повідомив: '%s'"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "Доступний новий реліз '%s' ."

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Виконайте команду 'do-release-upgrade' для оновлення."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""
"Перед модернізацією, будь ласка, встановіть усі наявні оновлення для вашого "
"випуску."

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""
"Ви не перезавантажилися після оновлення пакета, який цього потребує. Будь "
"ласка, перезавантажтесь перед модернізацією."

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr "Оновлювач програм"

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Доступне оновлення Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Ви відмовились від оновлення до Ubuntu %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Перевірити можливість оновлення до останнього випуску, що перебуває в стані "
"розробки"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "Додати результати відладки"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Виникла невиправна помилка при розрахунку оновлення:\n"
#~ "%s\n"
#~ "\n"
#~ " Причиною можуть бути:\n"
#~ " * Оновлення Ubuntu до версії pre-release\n"
#~ " * Використання поточної pre-release-версії Ubuntu\n"
#~ " * Неофіційні пакунки програм, що не підтримуються Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "Ваша система не містить пакунків ubuntu-desktop, kubuntu-deskto, xubuntu-"
#~ "desktop або edubuntu-desktop, через що не вдалося встановити, яку версію "
#~ "Ubuntu ви використовуєте.\n"
#~ " Будь ласка, спочатку встановіть один з цих пакунків, використовуючи "
#~ "synaptic або apt-get."

#~ msgid "Sandbox setup failed"
#~ msgstr "Помилка встановлення в пісочниці"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "Не вдалося створити середовище для пісочниці"

#~ msgid "Sandbox mode"
#~ msgstr "Режим пісочниці"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "Це оновлення запускається у режимі пісочниці (випробовуваному). Всі зміни "
#~ "записуються у '%s' і будуть втрачені при наступному перезапуску.\n"
#~ "\n"
#~ "*Ні* зміни записані до системної директорії з цього моменту до наступного "
#~ "перезапуску будуть постійні."

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "Ваше встановлення пакету python пошкоджене. Виправте посилання '/usr/bin/"
#~ "python'."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "Пакунок 'debsig-verify' встановлено"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "Оновлення не може бути продовжене, якщо цей пакунок встановлено.\n"
#~ "Будь ласка, спочатку вилучіть його за домогою synaptic або командою 'apt-"
#~ "get remove debsig-verify' та спробуйте запустити оновлення знову."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Під час сканування інформації у репозиторію не було знайдено запису про "
#~ "дзеркало для оновлення. Це може трапитися, якщо ви використовуєте "
#~ "внутрішнє дзеркало або інформація про дзеркала застаріла.\n"
#~ "\n"
#~ "Ви все ж таки бажаєте перезаписати 'sources.list'? Якщо ви оберете 'Так', "
#~ "то це оновить усі '%s' з '%s' записи.\n"
#~ "Якщо ви оберете 'Ні' оновлення не відбудеться."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "Оновлення було перервано. Для оновлення необхідно %s вільного місця на "
#~ "диску '%s'. Будь ласка, звільніть додатково хоча б %s дискового простору "
#~ "на '%s'. Очистіть кошик та вилучіть тимчасові пакунки з попередніх "
#~ "установок, використовуючи 'sudo apt-get clean'."

#~ msgid "evms in use"
#~ msgstr "використовується evms"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "Ваша система використовує менеджер томів 'evms' в /proc/mounts. Програма "
#~ "'evms' більше не підтримується, будь ласка, вимкніть її та запустіть "
#~ "оновлення знову."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Цей комп'ютер зараз використовує графічний драйвер NVIDIA 'nvidia'. Нема "
#~ "доступних версій драйверів, які б працювали з вашим обладнанням на Ubuntu "
#~ "10.04 LTS.\n"
#~ "\n"
#~ "Чи хочете ви продовжувати?"

#~ msgid "No i686 CPU"
#~ msgstr "Немає процесора i686"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "У вашому комп’ютері стоїть процесор i586 або інший, що не має розширення "
#~ "'cmov'. Усі ж пакети було зібрано з оптимізацією, що вимагає мінімум i686-"
#~ "архітектуру. Неможливо оновити систему до нової версії Ubuntu на цьому "
#~ "комп’ютері."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Оновлення Sandbox з використанням aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Протестувати оновлення в безпечному режимі"

#~ msgid "No new release found"
#~ msgstr "Нової версії не знайдено"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Невирішені проблеми під час розрахунку оновлення.\n"
#~ "\n"
#~ " Це могло бути викликано наступним:\n"
#~ " * Оновлення до версії Ubuntu, яка ще не вийшла офіційно\n"
#~ " * Запуск версії Ubuntu, яка ще не вийшла\n"
#~ " * Пакети неофіційного програмного забезпечення не постачаються Ubuntu\n"
#~ "\n"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Оновлюємо Ubuntu до версії 15.04</big></b>"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.10</big></b>"
#~ msgstr "<b><big>Оновлюємо Ubuntu до версії 15.10</big></b>"

#~ msgid "apt architecture is i386"
#~ msgstr "архітектурою apt є i386"

#~ msgid ""
#~ "Upgrades from 18.04 on the i386 architecture are not supported at this "
#~ "time."
#~ msgstr "Оновлення 18.04 на архітектурі i386 наразі не підтримуються."
